src/game/client/components/players.cpp

```diff
@@ -618,6 +618,13 @@ void CPlayers::RenderPlayer(
 	}
 }
 
+inline bool CPlayers::IsPlayerInfoAvailable(int ClientID)
```

Comment:
I think you can make this method const too

```suggestion
inline bool CPlayers::IsPlayerInfoAvailable(int ClientID) const
```